src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -124,11 +126,14 @@ class SceneryWindow final : public Window
             return std::find(std::begin(Entries), std::end(Entries), entry) != std::end(Entries);
         }
 
-        void AddEntry(const ScenerySelection& entry)
+        void AddEntry(const ScenerySelection& entry, bool toBack)
```

Comment:
just make two functions AddEntryFront and AddEntryBack true/false doesn't mean much and makes the code harder to read.

Reply:
This should be resolved now